src/main/java/com/webperside/courseerpbackend/models/mybatis/teacher/TeachersSchedule.java

```diff
@@ -0,0 +1,22 @@
+package com.webperside.courseerpbackend.models.mybatis.teacher;
+
+
+import com.webperside.courseerpbackend.models.mybatis.base.BaseEntity;
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDateTime;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TeachersSchedule extends BaseEntity<Long> {
+    Long branchId;
+    String dayOfWeek;
```

Comment:
Use DayOfWeek enum here

Reply:
Changes added according to your review. Please consider it again, thanks